.github/workflows/no-response.yaml

```diff
@@ -0,0 +1,40 @@
+name: No Response
+inputs:
+  # Comment to post when closing an Issue for lack of response. Set to `false` to disable
+  closeComment: >
```

Comment:
Does this work? The example has:
```yml
  closeComment:
    description: Comment to post when closing an Issue for lack of response. Set to `false` to disable.
    default: >
      This issue has been automatically closed because there has been no response
      to our request for more information from the original author. With only the
      information that is currently in the issue, we don't have enough information
      to take action. Please reach out if you have or find the answers we need so
      that we can investigate further.
```
https://github.com/lee-dohm/no-response/blob/main/action.yml

Reply:
I could not find a way to test it. Action.yaml is actually the action definition rather than an example. I'll monitor the change and revert it if it fails.